frameworks/digital-outcomes-and-specialists/questions/brief-responses/newAvailabilityOutcomes.yml

```diff
@@ -0,0 +1,19 @@
+name: Earliest start date
+id: availability
+question: When is the earliest the team can start
```

Comment:
Seems like it might be a good idea if the default slug used the ID, rather than the name?